releases/1.4/mlflow.yaml

```diff
@@ -0,0 +1,14 @@
+bundle: kubernetes
+name: mlflow-overlay
+applications:
+  mlflow-server: { charm: mlflow-server, channel: 1.13.1/stable, scale: 1 }
+  mlflow-db:
+    {
+      charm: charmed-osm-mariadb-k8s,
+      channel: latest/stable,
+      scale: 1,
+      options: { database: mlflow },
+    }
+relations:
+  - [mlflow-server, mlflow-db]
+  - [mlflow-server, minio]
```

Comment:
Are we missing an istio relation here?

Reply:
Do we include an istio relation here if we haven't deployed istio in the bundle?  Is our bundle meant to deliver exactly what mlflow needs and no more, or to deliver mlflow behind an existing istio?  I'd have expected just delivering mlflow, and the user could add istio on top

Reply:
I envision this as an overlay on top of kubeflow bundle. So I think it's fine to add the relation

Reply:
Yes correct, this is intended as an overlay to the kubeflow bundle, therefore istio should be there and we can add the relation.